server/routerlicious/packages/services-shared/src/runner.ts

```diff
@@ -25,7 +26,12 @@ export async function run<T extends IResources>(
 	resourceFactory: IResourcesFactory<T>,
 	runnerFactory: IRunnerFactory<T>,
 	logger: ILogger | undefined,
+	secretsList?: string[],
 ) {
+	if (config.get("config:configDumpEnabled")) {
```

Comment:
Maybe we can extract this to a different line to assign a default value if this is not defined in the config? Something like:

```ts
const configDumpEnabled = (config.get("config:configDumpEnabled") as boolean) ?? false;
if (configDumpEnabled) {
...
}
```